src/dune

```diff
@@ -0,0 +1,5 @@
+(library
+ (name note)
+ (public_name note)
+ (flags :standard -w -27-32-50)
```

Comment:
Please do not modify the warnings, we're trying to mess with the code as little as possible and for the installation with opam/opam-monorepo the code will be installed in release profile anyway.